thrift/src/main/thrift/contentatom.thrift

```diff
@@ -61,6 +61,7 @@ struct ContentChangeDetails {
 struct Flags {
 //1: optional bool suppressFurniture DEPRECATED
   2: optional bool legallySensitive
+  3: optional bool adsDisabled
```

Comment:
should it be `blockAds` to be consistent with [Composer](https://github.com/guardian/flexible-content/blob/master/flexible-content-api/src/main/scala/com/gu/flexiblecontent/api/event/stream/model/StreamableContent.scala#L198)?